### PR TITLE
Allow to mimic the normal cli behavior of docker for docker_user

### LINKED ISSUE #12: Why not use docker_user to administer docker
Hey, thanks for the role again. I was just in the process of creating a PR which adds a .bashrc to the docker_user, when I saw the note in your `README.md`: 
> Note that the sole purpose of the docker_user is to run the Docker daemon and related containers, and not for system administration or used as a regular user.

It somehow feels strange to me to use sudo every time, I run the docker command. Because I want docker to not have admin privileges. So my idea was to just switch into the user (`sudo su docker_user`) and than run commands as if it was rootful.

Why  are you arguing against such a use case? Would you still accept a PR with an optional .bashrc?

```
$ whoami
dockeruser

$ cat .bashrc 
export XDG_RUNTIME_DIR="/run/user/1002"
export DOCKER_HOST="unix:///run/user/1002/docker.sock"
export PATH="~/bin:$PATH"
```

### PR DESCRIPTION
**Pull request by t2d**
* Add new variable `docker_user_bashrc`, default false
* If true:
  * Manually download bash-completion
  * Deploy .bashrc for docker_user

Fix #12

### CODE REVIEW

**Comment by slhck** [comment]
Any news on this? I think this would be nice to have. Seems this was auto-closed for being stale.

**Comment by konstruktoid** [comment]
@t2d ?

**Comment by t2d** [comment]
I had created an additional role since this wasn't merged. Anyhow, here is a rebased patch. Hope it suits your needs

**Comment by konstruktoid** [comment]
It wasn't merged since none of my comments were addressed

**Comment by t2d** [comment]
What comment is that? I only see https://github.com/konstruktoid/ansible-docker-rootless/issues/12#issuecomment-894145035 which is basically against this PR as a whole. Therefore, I didn't follow up. But really, no bad feelings from my side. Take whatever you like and throw the rest away. Thanks for the nice role.

**Comment by konstruktoid** [comment]
I was referring to the review comments, but I'll have another look at the PR

image

**Comment by slhck** [comment]
Have you submitted them? These comments look pending to me. There is no review open on this PR, actually.

FWIW, I would prefer to have these lines added to the `.bashrc` via a block managed by Ansible rather than completely overwriting the file. Otherwise this overwrites some OS defaults (part of `/etc/skel/.bashrc`).

**Review by konstruktoid — CHANGES_REQUESTED** [review]

**Comment by konstruktoid** [comment]
> Have you submitted them? These comments look pending to me. There is no review open on this PR, actually.

Such fail from my side, sorry @t2d

**Comment by konstruktoid** [comment]
Thanks for the updates @t2d

**Review by konstruktoid — APPROVED** [review]

**Comment by konstruktoid** [comment]
I'll add a couple of test to this PR if that's ok.

File: README.md [hunk 1]
```diff
@@ -91,6 +93,11 @@ or `.bash_aliases`, otherwise a shell script named `docker_rootless.sh` is
 created in the Ansible user home directory that works as a substitute to the
 `docker` command.
 
+If `docker_compose: true`, then `docker-compose` will be installed via pip.
+
+If `docker_user_bashrc: true`, a .bashrc with completion for the docker(-compose)
+command will be place inside the `docker_user` home.
```
Comment (slhck) [inline README.md:99]: Minor nit: "placed"

**Comment by slhck** [comment]
Cool, thanks for both of your time! 👍

**Comment by konstruktoid** [comment]
merged.
and so sorry for the delay yet again.

will add tests during the day.

**Pull request merged**